src/scene/title_scene.rs

```diff
@@ -361,6 +361,25 @@ impl Scene for TitleScene {
             );
 
             batch.draw(ctx)?;
+        } else {
+            let selected_menu = self.current_menu;
+            let window_title = match selected_menu {
+                CurrentMenu::ChallengesMenu => (state.t("menus.main_menu.challenges")),
+                CurrentMenu::ChallengeConfirmMenu | CurrentMenu::SaveSelectMenu => (state.t("menus.main_menu.start")),
+                CurrentMenu::OptionMenu => (state.t("menus.main_menu.options")),
+                _ => "OOPS!".to_string(),
```

Comment:
This should throw a proper `unreachable!()` rather than nonsense text (since it should be unreachable)